app/assets/stylesheets/application.scss

```diff
@@ -40,6 +41,14 @@ section {
   height: 240px;
   width: 240px;
 }
+.sizeFramePhotos {
```

Comment:
Thường thì mình tránh đặt CSS vào trong file application.scss (dù việc đó không gây ra lỗi). Tốt hơn là nên tạo 1 file style riêng. Trường hợp style đó áp dụng cho toàn app thì có thể đặt tên file như globals.scss hoặc styles.scss